django-stubs/contrib/auth/base_user.pyi

```diff
@@ -13,6 +13,8 @@ class AbstractBaseUser(models.Model):
     last_login: Optional[models.DateTimeField] = ...
     is_active: models.BooleanField = ...
     REQUIRED_FIELDS: List[str] = ...
+    class Meta:
+        abstract: bool = True
```

Comment:
Replace `True` with `...` / remove `Meta` completely, please. Value will be specified in the actual sources, no need for that here.

Reply:
Yeah, I felt a bit weird putting those in but I haven't figured out how to get the value from the actual code when a stub is present. If you can point me in the right direction I'll gladly update the code and remove those from the stubs.